### DISCUSSION TITLE
clean target

### DISCUSSION THREAD

**Discussion by AlexeySmolenchuk** [microsoft/vscode-cmake-tools#2930] [Ideas]
It would be great to have the ability to run an automatic clean target from vscode interface.

**Comment by bobbrow** [comment]
What did you have in mind when you say "automatic"?  Are you just looking for UI that lets you start the clean?

**Comment by AlexeySmolenchuk** [comment]
Yes, exactly.
As I understood since CMake 3.0 "clean" target should be available automatically. But I don't see it in a list of all available targets.
